src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/oppgave/OppgaveClient.kt

```diff
@@ -62,8 +62,8 @@ class OppgaveClient(
         return getForEntity<Oppgave>(uri, uriVariables = oppgaveIdUriVariables(oppgaveId))
     }
 
-    fun hentOppgaver(finnOppgaveRequest: FinnOppgaveRequest): FinnOppgaveResponseDto {
-        val uri = UriComponentsBuilder.fromUri(oppgaveUri).pathSegment("finn").toUriString()
+    fun hentOppgaver(finnOppgaveRequest: FinnOppgaveRequestV2): FinnOppgaveResponseDto {
+        val uri = UriComponentsBuilder.fromUri(oppgaveUri).pathSegment("finnV2").toUriString()
```

Comment:
Hvis vi skal ha en `v2` av endepunktet så er det vel et poeng å beholde `v1` i en overgangsperiode? Ellers skjønner jeg ikke helt poenget med versjoneringen 😇

Reply:
Det burde jeg forklart tydligere i PR-en, men versjoneringen er mest for endringen integrasjoner. I integrasjoner har jeg lagt til et nytt endepunkt som forventer FinnOppgaveRequestV2. Siden vi ikke kan deploye integrasjoner og sak helt samtidig har jeg valgt å versjonere så vi kan deploye integrasjoner først, som tilbyr to endepunkter. Så deploye sak som tar i bruk det nye endepunktet. Da trenger ikke sak å bruke det gamle endepunktet lengere. 
https://github.com/navikt/tilleggsstonader-integrasjoner/pull/164

Reply:
Da henger jeg med!